ts/editor/helpers.ts

```diff
@@ -19,3 +22,13 @@ export function withFontColor(
 
     return false;
 }
+
+/***
+ * Required for field inputs wrapped in Collapsible
+ */
+export async function refocusInput(input: RichTextInput | PlainTextInput) {
```

Comment:
In the end you're just using `.api`, which is an object of type `EditingInputAPI`.

Reply:
Thanks! Why does the check fail though? I've added the return type :eyes:

Reply:
`api` is of type `EditingInputAPI`, not the entire RichTextInput / PlainTextInput

Reply:
Yes, but I already change that with https://github.com/ankitects/anki/pull/1990/commits/87fb5f6c7c32f0deb5fc69a78e7f022b82cb3e8e...

Reply:
Oh sorry i got it. I'll fix that once I'm home.

Reply:
Turns out I didn't get it after all. TS is still uncharted terrain for me. I don't know why the function would expect an argument of type `EditingArea` if I typed it as `EditingInputAPI`.